### PR TITLE
markersWatchOptions, directiveControls

### PR DESCRIPTION
**Pull request by nmccready**
ability to disable watches in markers 100%

markersControl setup to be fetched from leafletData.directivesControls

### CODE REVIEW

File: examples/overlays-markers-nested-example-no-watch.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+                    marker.lat += .05;
+                }
+            }
+            //force manual update
+            $scope.markers = _clonedMarkers;
+            controls.markers.create($scope.markers);
+        });
+    }, 4000);
+    angular.extend($scope, {
+        markerWatchOptions: {
```
Comment (nmccready) [inline examples/overlays-markers-nested-example-no-watch.html]: disable watches example

File: examples/overlays-markers-nested-example-no-watch.html [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+            for (var layer in _clonedMarkers) {
+                var markerSet = _clonedMarkers[layer];
+                for (var markerName in markerSet) {
+                    var marker = markerSet[markerName];
+                    marker.lat += .05;
+                }
+            }
+            //force manual update
+            $scope.markers = _clonedMarkers;
+            controls.markers.create($scope.markers);
```
Comment (nmccready) [inline examples/overlays-markers-nested-example-no-watch.html]: manual update

File: src/directives/markers.js [hunk 3]
```diff
@@ -1,10 +1,9 @@
 angular.module("leaflet-directive").directive('markers',
-    function ($log, $rootScope, $q, leafletData, leafletHelpers, leafletMapDefaults, leafletMarkersHelpers,
-              leafletEvents, leafletIterators) {
+    function ($log, $rootScope, $q, $parse, leafletData, leafletHelpers,
```
Comment (nmccready) [inline src/directives/markers.js]: need to remove $parse as it is not used

File: src/services/leafletData.js [hunk 4]
```diff
@@ -3,112 +3,48 @@ angular.module("leaflet-directive").service('leafletData', function ($log, $q, l
         getUnresolvedDefer = leafletHelpers.getUnresolvedDefer,
         setResolvedDefer = leafletHelpers.setResolvedDefer;
```
Comment (nmccready) [inline src/services/leafletData.js]: simplify the redundancy of adding more data

**Comment by nmccready** [comment]
@tombatossals @zacronos

**Comment by tombatossals** [comment]
Great job @nmccready, merged

**Pull request merged**